include/zephyr/fs/fcb.h

```diff
@@ -129,6 +135,11 @@ struct fcb {
 	/**< The value flash takes when it is erased. This is read from
 	 * flash parameters and initialized upon call to fcb_init.
 	 */
+
+#if IS_ENABLED(CONFIG_FCB_ALLOW_FIXED_ENDMARKER)
```

Comment:
In include files you can use `#ifdef`.
In c files where it is used as part of a routine it is better to use `if IS_ENABLED()` to improve readability of your code